src/lib/Form/Data/Content/CustomUrl/CustomUrlAddData.php

```diff
@@ -28,18 +28,23 @@ class CustomUrlAddData
     /** @var bool */
     private $siteRoot;
 
+    /** @var int|null */
+    private $rootLocationId;
+
     public function __construct(
         ?Location $location = null,
         ?string $path = null,
         ?Language $language = null,
         bool $redirect = true,
-        bool $siteRoot = true
+        bool $siteRoot = true,
+        ?int $rootLocationId = null
     ) {
         $this->location = $location;
         $this->path = $path;
         $this->language = $language;
         $this->redirect = $redirect;
         $this->siteRoot = $siteRoot;
+        $this->$rootLocationId = $rootLocationId;
```

Comment:
Looks like a typo. Please add test coverage to avoid those, they're difficult to spot.

Reply:
> Why was this added via listener instead directly to a form? Don't know much about AdminUI architecture...

I guess in order to keep it clean. In this form type there are 4 other listeners working exactly the same.

Reply:
Fixed the typo, however, not sure if we test the form data anywhere in `admin-ui`.

Reply:
> Fixed the typo, however, not sure if we test the form data anywhere in `admin-ui`.

a new test case for `CustomUrl/CustomUrlAddData`, for the new methods only, wouldn't hurt. It could be generated by PHPStorm in less than a minute, though probably proper namespace needs to be set.

Reply:
@alongosz resolved in https://github.com/ezsystems/ezplatform-admin-ui/pull/1583/commits/0a598d578df2ae05c08ee8c40145daac086fe23a